Cubical/HITs/Pushout/Properties.agda

```diff
@@ -0,0 +1,209 @@
+                        (inS (push (push a j) i))
+
+  A○□→A□○ : A○□ → A□○
+  A○□→A□○ (inl x) = backward-l x
+  A○□→A□○ (inr x) = backward-r x
+  A○□→A□○ (push (inl x) i) = inl (push x i)
+  A○□→A□○ (push (inr x) i) = inr (push x i)
+  A○□→A□○ (push (push a i) j) = backward-filler a i j i1
+
+  homotopy1-l : ∀ x → A□○→A○□ (backward-l x) ≡ inl x
+  homotopy1-l (inl x) = refl
+  homotopy1-l (inr x) = refl
+  homotopy1-l (push a i) = refl
+
+  homotopy1-r : ∀ x → A□○→A○□ (backward-r x) ≡ inr x
+  homotopy1-r (inl x) = refl
+  homotopy1-r (inr x) = refl
+  homotopy1-r (push a i) = refl
+
+  A○□→A□○→A○□ : ∀ x → A□○→A○□ (A○□→A□○ x) ≡ x
+  A○□→A□○→A○□ (inl x) = homotopy1-l x
+  A○□→A□○→A○□ (inr x) = homotopy1-r x
+  A○□→A□○→A○□ (push (inl x) i) k = push (inl x) i
+  A○□→A□○→A○□ (push (inr x) i) k = push (inr x) i
+  A○□→A□○→A○□ (push (push a i) j) k =
+    hcomp (λ t → λ { (i = i0) → forward-l (doubleCompPath-filler (λ k → inl (H11 a k)) (push (f21 a)) (λ k → inr (H31 a (~ k))) j (~ t))
+                   ; (i = i1) → forward-r (doubleCompPath-filler (λ k → inl (H13 a k)) (push (f23 a)) (λ k → inr (H33 a (~ k))) j (~ t))
+                   ; (j = i0) → homotopy1-l (doubleCompPath-filler (λ k → inl (H11 a (~ k))) (push (f12 a)) (λ k → inr (H13 a k)) i t) k
+                   ; (j = i1) → homotopy1-r (doubleCompPath-filler (λ k → inl (H31 a (~ k))) (push (f32 a)) (λ k → inr (H33 a k)) i t) k
+                   ; (k = i0) → A□○→A○□ (backward-filler a i j t)
+                   ; (k = i1) → forward-filler a j i (~ t) })
+          (forward-filler a j i i1)
+
+  homotopy2-l : ∀ x → A○□→A□○ (forward-l x) ≡ inl x
+  homotopy2-l (inl x) = refl
+  homotopy2-l (inr x) = refl
+  homotopy2-l (push a i) = refl
+
+  homotopy2-r : ∀ x → A○□→A□○ (forward-r x) ≡ inr x
+  homotopy2-r (inl x) = refl
+  homotopy2-r (inr x) = refl
+  homotopy2-r (push a i) = refl
+
+  A□○→A○□→A□○ : ∀ x → A○□→A□○ (A□○→A○□ x) ≡ x
+  A□○→A○□→A□○ (inl x) = homotopy2-l x
+  A□○→A○□→A□○ (inr x) = homotopy2-r x
+  A□○→A○□→A□○ (push (inl x) i) k = push (inl x) i
+  A□○→A○□→A□○ (push (inr x) i) k = push (inr x) i
+  A□○→A○□→A□○ (push (push a i) j) k =
+    hcomp (λ t → λ { (i = i0) → backward-l (doubleCompPath-filler (λ k → inl (H11 a (~ k))) (push (f12 a)) (λ k → inr (H13 a k)) j (~ t))
+                   ; (i = i1) → backward-r (doubleCompPath-filler (λ k → inl (H31 a (~ k))) (push (f32 a)) (λ k → inr (H33 a k)) j (~ t))
+                   ; (j = i0) → homotopy2-l (doubleCompPath-filler (λ k → inl (H11 a k)) (push (f21 a)) (λ k → inr (H31 a (~ k))) i t) k
+                   ; (j = i1) → homotopy2-r (doubleCompPath-filler (λ k → inl (H13 a k)) (push (f23 a)) (λ k → inr (H33 a (~ k))) i t) k
+                   ; (k = i0) → A○□→A□○ (forward-filler a i j t)
+                   ; (k = i1) → backward-filler a j i (~ t) })
+          (backward-filler a j i i1)
+
+  
+
+  3x3 : A□○ ≡ A○□
```

Comment:
Maybe call this `Pushout3x3` ?

Reply:
Could you also put this statement outside of the module?

Reply:
> Could you also put this statement outside of the module?
Do you mean something like what I added at the end of the file in commit 1109cc5 ? It does not look very pretty, but probably using abstractions like spans and transpose would make it better (as you already suggested)

Reply:
Yeah, packaging things up will make it look nicer. But let's keep it as it is for now